apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1003,163 +1003,184 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
+                    expiryInSeconds: Number(values.get("idExpiryInSeconds")),
+                    idTokenSignedResponseAlg: values.get("idTokenSignedResponseAlg") !== "None" ?
+                        values.get("idTokenSignedResponseAlg") : null
                 },
-                pkce: null,
-                refreshToken: null
-            };
-        }
-
-        inboundConfigFormValues = {
-            ...inboundConfigFormValues,
-            clientAuthentication: {
-                tlsClientAuthSubjectDn: subjectDN,
-                tokenEndpointAuthMethod: values.get("tokenEndpointAuthMethod") !== "None" ?
-                    values.get("tokenEndpointAuthMethod") : null,
-                tokenEndpointAuthSigningAlg: values.get("tokenEndpointAuthSigningAlg")
-            },
-            pushAuthorizationRequest: {
-                requirePushAuthorizationRequest: values.get("requirePushAuthorizationRequest")?.length > 0
-            },
-            requestObject: {
-                encryption: {
-                    algorithm: values.get("requestObjectEncryptionAlgorithm") !== "None" ?
-                        values.get("requestObjectEncryptionAlgorithm") : null,
-                    method: values.get("requestObjectEncryptionMethod") !== "None" ?
-                        values.get("requestObjectEncryptionMethod") : null
+                logout: {
+                    backChannelLogoutUrl: values.get("backChannelLogoutUrl"),
+                    frontChannelLogoutUrl: values.get("frontChannelLogoutUrl")
                 },
-                requestObjectSigningAlg: values.get("requestObjectSigningAlg") !== "None" ?
-                    values.get("requestObjectSigningAlg") : null
-            },
-            subject: {
-                sectorIdentifierUri: initialValues?.subject?.sectorIdentifierUri,
-                subjectType: initialValues?.subject?.subjectType
+                publicClient: !isMobileApplication ? values.get("supportPublicClients")?.length > 0 : true,
+                refreshToken: {
+                    expiryInSeconds: values.get("expiryInSeconds")
+                        ? parseInt(values.get("expiryInSeconds"), 10)
+                        : Number(metadata.defaultRefreshTokenExpiryTime),
+                    renewRefreshToken: values.get("RefreshToken")?.length > 0
+                },
+                scopeValidators: values.get("scopeValidator"),
+                validateRequestObjectSignature: values.get("enableRequestObjectSignatureValidation")?.length > 0
+            };
+    
+            !applicationConfig.inboundOIDCForm.showFrontChannelLogout
+                && delete inboundConfigFormValues.logout.frontChannelLogoutUrl;
+            !applicationConfig.inboundOIDCForm.showScopeValidators
+                && delete inboundConfigFormValues.scopeValidators;
+            !applicationConfig.inboundOIDCForm.showIdTokenEncryption
+            && delete inboundConfigFormValues.idToken.encryption;
```

Comment:
Indentation issue here. Let's check other places as well.
```suggestion
            !applicationConfig.inboundOIDCForm.showIdTokenEncryption
                && delete inboundConfigFormValues.idToken.encryption;
```